### DISCUSSION TITLE
How to use the Prism's different themes of code block in lotusdocs?

### DISCUSSION THREAD

**Discussion by SWHL** [colinwilson/lotusdocs#53] [Q&A] [answered]
I want to use the twilight code highlight theme of Prism in the lotusdocs, but I only see the only paramter `prism = true` in the `[param.docs]`.

If I want to change the theme of Code Highlighters, how can I do?

Thanks.

**Comment by colinwilson** [comment, marked as answer]
Prism themes aren't supported (yet!). A limited number of themes will be supported soon though.

For now, you can update / alter the [`_prism.scss`](https://github.com/colinwilson/lotusdocs/blob/release/assets/docs/scss/custom/plugins/prism/_prism.scss) file.

**Comment by SWHL** [comment]
Look forward to supporting other themes.

**Comment by colinwilson** [comment]
[PrismJs Themes!](https://lotusdocs.dev/docs/guides/features/syntax-highlighting/#themes) 👀

Will add more over time. 👍
